includes/class-wcpdf-main.php

```diff
@@ -290,7 +290,7 @@ public function generate_pdf_ajax() {
 		$order_ids = (array) array_map( 'absint', explode( 'x', $_GET['order_ids'] ) );
 
 		// Process oldest first: reverse $order_ids array if required
-		if ( count( $order_ids ) > 1 && end( $order_ids ) < reset( $order_ids ) ) {
+		if ( count( $order_ids ) > 1 && end( $order_ids ) < reset( $order_ids ) && apply_filters( 'wpo_wcpdf_pdf_ajax_reverse_order_ids', true ) ) {
```

Comment:
I think it's more intuitive to let the filter override sort order than to let the filter determine whether to reverse. Right now it sorts ASC by order ID, even if the sorting of the call was already ASC. So something like this:
```php
$sort_order = apply_filters( 'wpo_wcpdf_bulk_document_sort_order', 'ASC' );
$current_sort_order = ( count( $order_ids ) > 1 && end( $order_ids ) < reset( $order_ids ) ) ? 'DESC' : 'ASC';
if ( in_array( $sort_order, array( 'ASC', 'DESC' ) ) && $sort_order != $current_sort_order ) {
	$order_ids = array_reverse( $order_ids );
}
```

(_not_ tested!)